recipe/meta.yaml

```diff
@@ -23,9 +23,11 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake
     - perl
-    - nasm
-    - make   # [unix]
-
+    - nasm      # [x86_64]
+    - binutils  # [aarch64]
```

Comment:
We might want to add a comment to say that we need binutils for the `as` executable`.

Reply:
```suggestion
    # we need binutils for the `as` executable
    - binutils  # [aarch64]
```

Reply:
I will wait for the checks to finish, thanks!